tests/serialization/test_prior_serialization.py

```diff
@@ -0,0 +1,13 @@
+"""Test serialization of kernels."""
+
+from hypothesis import given
+
+from baybe.kernels.priors import GammaPrior
+from tests.hypothesis_strategies.priors import gamma_priors
+
+
+@given(gamma_priors)
+def test_gamma_prior_kernel_roundtrip(prior: GammaPrior):
```

Comment:
i know we only have gammapriors at the moment

but given youve already introduce a generic `priors` strategy why not make this test name and hypothesis general too? it would mean any update and addition of enw priors wouldnt require any change here anymore

Reply:
Upvote! This was exactly the motivation for the generic `priors` strategy: to avoid later changes in other places 👍🏼

Reply:
Changed. 

BTW: You realized stuff like this, but NOBODY saw that the module docstring said `"""Test serialization of kernels."""` :D

Reply:
Was supposed to be a test ... you passed it, congratulations 🤓

Reply:
i dont know why, but I always ignore titles of modules and powerpoint slides, they are simply irrelevant to me  🤷‍♂️